api/v1alpha1/terraform_types.go

```diff
@@ -168,6 +168,9 @@ type TerraformSpec struct {
 	// +kubebuilder:default:=false
 	// +optional
 	RefreshBeforeApply bool `json:"refreshBeforeApply,omitempty"`
+
+	// +optional
+	RunnerPod RunnerPod `json:"runnerPod,omitempty"`
```

Comment:
Sure I will update it

Reply:
Agreed with @phoban01, it would be nice to use `runnerPodSpec` or `runnerPodTemplate`.
Also, would you please add `image:` to this change too. This `image` setting would override image for each object.